Flank/src/main/java/com/walmart/otto/configurator/Configurator.java

```diff
@@ -24,7 +24,7 @@
   private boolean fetchXMLFiles = true;
   private boolean debug = false;
   private boolean fetchBucket = false;
-  private boolean disableAutoGoogleLogin = false;
+  private boolean autoGoogleLogin = false;
```

Comment:
The default for auto google login should be true to match the gcloud CLI.

> Automatically log into the test device using a preconfigured Google account before beginning the test. Enabled by default, use --no-auto-google-login to disable
https://cloud.google.com/sdk/gcloud/reference/firebase/test/android/run

Reply:
I made it false by default as there have been issues with it recently. Lemme know if you still want it changed.

Reply:
> I made it false by default as there have been issues with it recently. Lemme know if you still want it changed.

I think we should stay with gcloud defaults as a general principle. Google is rapidly fixing bugs so we'd be out of date with any custom changes.

Reply:
Done